Tests/Controller/HelperControllerTest.php

```diff
@@ -697,4 +720,23 @@ public function testRetrieveAutocompleteItemsActionNotGrantedTarget()
 
         $this->controller->retrieveAutocompleteItemsAction($request);
     }
+
+    /**
+     * Symfony Validator has 2 API version (2.4 and 2.5)
+     * This data provider ensure tests pass on each one.
+     */
+    public function getValidatorInterfaces()
+    {
+        $data = array();
+
+        $data['2.4'] = array('Symfony\Component\Validator\ValidatorInterface');
```

Comment:
For Symfony 3.0 compatibility I'd suggest this too be wrapped in an `interface_exists()` check since this interface is removed there.